chosen/chosen.css

```
@@ -1,6 +1,7 @@
 /* @group Base */
 .chzn-container {
-  font-size: 13px;
+  font-size: 16px;
+  font-weight: 600;
```

Too heavy! Let's use
```suggestion
  font-weight: 500;
```